core-domain/src/main/java/org/uniprot/core/uniprotkb/impl/UniProtKBEntryBuilder.java

```diff
@@ -254,6 +278,11 @@ private UniProtKBEntryBuilder(
         return this;
     }
 
+    public @Nonnull UniProtKBEntryBuilder extraAttributesAdd(String name, Object value) {
```

Comment:
we could use the enum as the name parameter to restrict it. (complement of my first comment in the constants lines)